README.md

```diff
@@ -172,6 +172,30 @@ $authorization = $client->authorizations->authorize([
 ]);
 ```
 
+## PSR-11 factory
+
+You can use the predefined factory `Clearhaus\Container\ClientFactory` to instantiate a Clearhaus client:
+
+```php
+use Clearhaus\Container\ClientFactory;
+
+$factory = new ClientFactory();
+$client = $factory($psrContainer);
+```
+
+The client configuration must look like below:
+
+```php
+return [
```

Comment:
Add `use Clearhaus\Client;` or add the FQCN when using `Client::MODE_TEST` below